packages/clerk-js/webpack.config.js

```diff
@@ -48,6 +48,16 @@ const common = ({ mode }) => {
     optimization: {
       splitChunks: {
         cacheGroups: {
+          zxcvbnTSCoreVendor: {
```

Comment:
⛔ Since we lazy load, these settings should be removed.

Reply:
If we do not create a separate cacheGroup, zxcvbn will be included in defaultVendors and we will end up loading the library eagerly.

Am i missing something ?

Reply:
This case is similar to https://github.com/clerkinc/javascript/blob/493ddc19f1fbbbaaa34bdbf5beaa15be9eeb75bc/packages/clerk-js/src/utils/url.ts#L67. Do we want webpack to coordinate the loading or do we want to do it ourselves?

cc: @nikosdouvlis

Reply:
I see, thank you Sokratis.
They do offer [cdn links](https://zxcvbn-ts.github.io/zxcvbn/guide/getting-started/#as-script-tag) that we can leverage.

Reply:
Great. PTAL https://github.com/clerkinc/javascript/pull/1050#discussion_r1161679217